easybuild/easyconfigs/b/Biopython/Biopython-1.65-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,43 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 The Cyprus Institute
+# Authors::   Andreas Panteli <a.panteli@cyi.ac.cy>, Thekla Loizou <t.loizou@cyi.ac.cy>, George Tsouloupas <g.tsouloupas@cyi.ac.cy>
+# License::   MIT/GPL
+#
+##
+easyblock = "PythonPackage"
+
+name = 'Biopython'
+version = '1.65'
+
+homepage = 'http://www.biopython.org'
+description = """Biopython is a set of freely available tools for biological computation written
+in Python by an international team of developers. It is a distributed collaborative effort to
+develop Python libraries and applications which address the needs of current and future work in
+bioinformatics. """
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://biopython.org/DIST']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+
+python = 'Python'
+pyver = '2.7.11'
+pyshortver = '.'.join(pyver.split('.')[:2])
```

Comment:
drop these three lines (and avoid introducing a double blank line please), no longer needed (cfr. changes to `dependencies`)